pkg/cmd/tknpac/resolve/resolve.go

```diff
@@ -204,17 +204,17 @@ func resolveFilenames(ctx context.Context, cs *params.Run, filenames []string, p
 	}
 
 	// TODO: flags
-	allTemplates = templates.ReplacePlaceHoldersVariables(allTemplates, params)
+	allTheYamls = templates.ReplacePlaceHoldersVariables(allTheYamls, params)
 	// We use github here but since we don't do remotetask we would not care
 	providerintf := github.New()
 	event := info.NewEvent()
-	prun, err := resolve.Resolve(ctx, cs, cs.Clients.Log, providerintf, event, allTemplates, ropt)
+	prun, err := resolve.Resolve(ctx, cs, cs.Clients.Log, providerintf, event, allTheYamls, ropt)
 	if err != nil {
 		return "", err
 	}
 
 	// cleanedup regexp do as much as we can but really it's a lost game to try this
-	cleanRe := regexp.MustCompile(`\n(\t|\s)*(creationTimestamp|spec|taskRunTemplate|metadata|computeResources):\s*(null|{})\n`)
+	cleanRe := regexp.MustCompile(`\n(\t|\s)*(status|taskRunTemplate|creationTimestamp|spec|taskRunTemplate|metadata|computeResources):\s*(null|{})\n`)
```

Comment:
Ho it use to handle till this time :thinking:

Reply:
what do you mean?

Reply:
it wasn't removing those useless fields before.... it's just useless fields made by the yaml struct dumper not harmless when doing pipelinerun creation